### ISSUE TITLE
The error is not reported as expected

### ISSUE THREAD

**Issue by severo** [huggingface/datasets-server#1115]
See https://github.com/huggingface/datasets-server/issues/1110, for example: 

the error, for the step `config-parquet-and-info`, is:

```
zstd decompress error: Frame requires too much memory for decoding
```

But we get:

```
list.extend() takes exactly one argument (7 given)
```

**Comment by severo** [comment]
Should be fixed by https://github.com/huggingface/datasets-server/pull/1116

**Comment by albertvillanova** [comment]
Closed by #1116
